configuration/rules-dsl.md

```diff
@@ -178,6 +179,27 @@ then
 end
 ```
 
+### Thing Triggers
+
+You can listen on status updates or on status changes (an update might leave the status unchanged).
+You can decide whether you want to catch only a specific status or any.
+Here is the syntax for all these cases (parts in square brackets are optional):
+```java
+Thing <ThingUid> received update [<status>]
+Thing <ThingUid> changed [from <status>] [to <status>]
+```
+
+The ThingUid is the identifier assigned by OpenHAB to the thing you add. You can find it from
+PaperUI. For example, one of my z-wave device is assigned "zwave:device:c5155aa4:node14".
```

Comment:
*openHAB

> You can find it from PaperUI.

This sentence is a bit vague. You can also get it by looking at the Karaf remote console.

> one of my z-wave device is assigned

There is no "I" in documentation. 😜